src/scanner.zig

```diff
@@ -154,18 +154,32 @@ const Protocol = struct {
                 if (mem.eql(u8, tag, "copyright")) {
                     if (copyright != null)
                         return error.DuplicateCopyright;
-                    const e = parser.next() orelse return error.UnexpectedEndOfFile;
-                    switch (e) {
-                        .character_data => |data| copyright = try allocator.dupe(u8, data),
-                        else => return error.BadCopyright,
+                    while (parser.next()) |e| {
+                        switch (e) {
+                            .character_data => |data| {
+                                copyright = try allocator.dupe(u8, data);
+                                break;
+                            },
+                            .attribute => continue,
+                            else => return error.BadCopyright,
+                        }
+                    } else {
+                        return error.UnexpectedEndOfFile;
```

Comment:
this change should be reverted, the copyright can't have any attributes: https://gitlab.freedesktop.org/wayland/wayland/-/blob/master/protocol/wayland.dtd